feat: added all required sub pages UI for the community section

AceTheCreator opened the pull request:
This PR shows all community sub-page's user interfaces which includes
- /community/ambassador-programs
- /community/contributors
- /community/events

AceTheCreator committed:
extended annoucement banner functionality

AceTheCreator committed:
importing changes

AceTheCreator committed:
added ng images

AceTheCreator committed:
adding ambassador program section

AceTheCreator committed:
implementing ambassador page

AceTheCreator committed:
adding program section

AceTheCreator committed:
completed first iteration of ambassador page

AceTheCreator committed:
enhancing page view

AceTheCreator committed:
added the last section of ambassador page

AceTheCreator committed:
enhanced speakers card

AceTheCreator committed:
made page responsive

AceTheCreator committed:
addtions

AceTheCreator committed:
Merge branch 'community' into communityDev

AceTheCreator committed:
added link to ambassador page

AceTheCreator committed:
added contributors route

AceTheCreator commented:
@derberg why's Netlify failing?

derberg commented:
@AceTheCreator I only know what is in the logs:

```
1:25:09 PM: Failed to compile.
1:25:09 PM: 
1:25:09 PM: ./node_modules/@asyncapi/modelina/lib/esm/helpers/FileHelpers.js
1:25:09 PM: Module not found: Can't resolve 'fs' in '/opt/build/repo/node_modules/@asyncapi/modelina/lib/esm/helpers'
1:25:09 PM: Import trace for requested module:
1:25:09 PM: ./node_modules/@asyncapi/modelina/lib/esm/helpers/index.js
1:25:09 PM: ./node_modules/@asyncapi/modelina/lib/esm/index.js
1:25:09 PM: ./components/modelina/TypeScriptGenerator.js
1:25:09 PM: ./pages/tools/modelina.js
1:25:09 PM: > Build failed because of webpack errors
```

but why all suddenly there is a problem with modelina, I have no idea 😄 you need to investigate 😄

AceTheCreator committed:
added contribors page with dummy data

AceTheCreator committed:
creating contributors sub pages

AceTheCreator committed:
added the subpages for contributors

AceTheCreator committed:
made contributors page responsive

AceTheCreator committed:
feat: new community section (#902)

AceTheCreator committed:
Merge branch 'community' of https://github.com/asyncapi/website into communityDev

AceTheCreator committed:
Completed all required pages

AceTheCreator committed:
Fixed fs issue with modelina

AceTheCreator commented:
Added additional pages UI that are ready for preview cc @derberg @akshatnema

AceTheCreator commented:
@Barbanio @starlightknown Please you folks should kindly review the ambassador program page UI i added to this PR :)

quetzalliwrites commented:
Aaaaaaaaaaaaah, why so many pics of me?!?! O_O

<img width="250" alt="Screen Shot 2022-11-07 at 1 17 27 PM" src="https://user-images.githubusercontent.com/19964402/200417389-46f69750-9d40-44c5-b097-58b62862f425.png">

quetzalliwrites commented:
Ok, jokes aside here's the main problem I still see in this PR. 

## `Community Docs` have no differentiation from the **AsyncAPI Docs**
Question: When are we adding written content to the Community Docs? I would not approve merging this PR unless docs existed. 

1. These docs will confuse our users because they look like a mistake copied over from the AsyncAPI Docs. Instead of only writing a title of *Docs*, you should prob change it to `Community Docs`.
2. Why is the following text written in the **Community Docs** homepage? _"Explore our API documentation, developer tools, and integration resources."_ That is wrong, ALL docs about the technology do not belong in Community Docs but in the actual AsyncAPI Docs. I am starting to get concerned with the weird duplication effect these Community Docs have.
3. The `Getting Started` section looks like a mistaken duplicate of our current [AsyncAPI Docs Quickstart, "Getting Started."](https://www.asyncapi.com/docs/tutorials/getting-started) I do not think this title should be used for this section of the Community Docs. What is the purpose of this section?
<img width="500" alt="Screen Shot 2022-11-07 at 1 20 02 PM" src="https://user-images.githubusercontent.com/19964402/200418073-31e30bd6-f204-4988-b968-cbb5febac44c.png">

<img width="300" alt="Screen Shot 2022-11-07 at 1 19 40 PM" src="https://user-images.githubusercontent.com/19964402/200418062-d97fd2a1-ee41-4855-bd95-8b19afee3427.png">


## The last section of 3 cards still do not click through
This section should not be added UNLESS those cards are changed to be clickable.

<img width="400" alt="Screen Shot 2022-11-07 at 1 26 08 PM" src="https://user-images.githubusercontent.com/19964402/200418896-627a438e-f6f1-4477-a97e-05e47ddd60b1.png">

quetzalliwrites reviewed (changes requested):
Left my review, still items to address and think about before we can get closer to merging this work.

AceTheCreator committed:
Added links to community main page

AceTheCreator committed:
updated the community docs link

AceTheCreator commented:
> 



> Ok, jokes aside here's the main problem I still see in this PR.
> 
> ## `Community Docs` have no differentiation from the **AsyncAPI Docs**
> Question: When are we adding written content to the Community Docs? I would not approve merging this PR unless docs existed.
> 

I'm aware of that. And the purpose of this PR is to review the first iteration of the implementation of the community section. The docs will eventually come in before the final review

AceTheCreator commented:
> 2. Why is the following text written in the **Community Docs** homepage? _"Explore our API documentation, developer tools, and integration resources."_ That is wrong, ALL docs about the technology do not belong in Community Docs but in the actual AsyncAPI Docs. I am starting to get concerned with the weird duplication effect these Community Docs have.
> 

One of the main purposes of this PR is so that you folks will be able to suggest content replacements. I literally just added em in place on some dummy text.

AceTheCreator commented:
> 3. The `Getting Started` section looks like a mistaken duplicate of our current [AsyncAPI Docs Quickstart, "Getting Started."](https://www.asyncapi.com/docs/tutorials/getting-started) I do not think this title should be used for this section of the Community Docs. What is the purpose of this section?
> 

The basis of this section is to contain docs like "How to become a TSC member", "How to create a new repository in the AsyncAPI organization" e.t.c

akshatnema commented on components/JobSummary.js:
```diff
@@ -1,45 +1,99 @@
+          <dl>
+            <dt className="sr-only">Employment type</dt>
+            <dd className="flex mt-1 text-sm text-gray-900">
+              <svg
```
I'm not sure everyone will be happy on my suggestion but I think that the SVG you are adding, you shouldn't add the SVG code directly in the file because it decreases code readability as I'm not able to recognise which SVG it is pointing to. If you want to add Tailwind classes inside it, I will suggest having their components inside the `icons` folder and importing them with the custom name you provide to the component. You already did this for many icons as I can see in the PR.

AceTheCreator replied:
This review is kinda not related to the PR :( You can open an issue for that

akshatnema replied:
But, this issue is not in the official documentation of the AsyncAPI in the website. Maybe there's just some small error you missed while making the Docs Nav, and this can be fixed using the reference of Docs Nav in documentation.

AceTheCreator replied:
Docs Nav? I'm typically reusing the existing one

akshatnema commented on components/layout/DocsLayout.js:
```diff
@@ -142,11 +142,11 @@ export default function DocsLayout({ post, navItems = {}, children }) {
               
               <nav className="flex-1 bg-white">
-                <ul>
+                <u l>
```
This change is quite confusing to me because, in this way, I think you are allowing to have dark points beside each Nav option in Docs and it is not correct in terms of my view. I'll still ask for opinions from @derberg and @alequetzalli.

![image](https://user-images.githubusercontent.com/76521428/200585739-fa25a1bd-c9c2-4fb6-af85-3e8baf658fd1.png)

quetzalliwrites replied:
Agreed, we should NOT have `<li>` bullet points showing.

AceTheCreator replied:
Fixed!

akshatnema commented on pages/community/ambassador-program/index.js:
```diff
@@ -0,0 +1,305 @@
+import IconRocket from '../../../components/icons/Rocket';
+import NewsletterSubscribe from '../../../components/NewsletterSubscribe';
+
+const data = [
```
Please make these array variables in a separate file and import it from there so that if we update/add more elements to it, it won't clutter the main Dashboard implementation from the values we are using as arrays.

AceTheCreator replied:
Please ignore this kind of review for now. The reason for this is just for quick prototyping :)

akshatnema replied:
But if we ignore it right now and move these changes for final review, it will be very much difficult for me to review around 3k-4k lines of code in terms of changes, so it's better if we do the reviews in small PRs also, so that the burden for final PR won't be much.

akshatnema commented on pages/community/index.js:
```diff
@@ -440,43 +493,20 @@ function CommunityIndexPage() {
-          <div className="h-half-screen md:h-full w-full md:w-2/4 md:flex md:flex-col md:justify-around mt-10 md:mt-0">
-            <div className="h-4/6 md:h-beforeHalf rounded-lg shadow-lg bg-usecases bg-cover bg-center relative">
-              <div className="w-full h-full bg-pink-500 opacity-25 absolute rounded-lg" />
+            <a href="/docs/tutorials">
```
This should be a `Link` tag from `next/link` instead of `a` tag.

AceTheCreator replied:
Lol. I prefer using this method instead of next/link for external link. Is there a specific reason why you think i should change it?

akshatnema replied:
Yeah, because `Link` tag provides better routing facility than `a` tag in Next js. For more details, you can visit: 

https://nextjs.org/docs/routing/introduction#linking-between-pages

AceTheCreator replied:
Oh! sorry i thought you were talking about linking to a different domain entirely

akshatnema reviewed (changes requested):
Brilliant contributions @AceTheCreator 👏 , but also I would like to add few changes that I consider to be important. Right now, I have made review on the basis of UI preview, but will do the review in terms of code addition and implementation afterwards (as soon as I get free time). Here are some of the issues I'll like to entertain:

### Not having proper color or background addition for cards

![image](https://user-images.githubusercontent.com/76521428/200588453-6fbc5a4b-033d-4f10-ac77-ba4a6b2da070.png)

Tbh, if you are assuming them to be cards, you should have either a good background inside the each card or they should have a slight wide border so that user can identify that these cards are clickable and different from the background.

### Buttons are not exactly in the way we use in website

![image](https://user-images.githubusercontent.com/76521428/200588993-229a242a-1462-45f5-a7c5-3e39b711bb04.png)

If we look into the Buttons you added in the Community Dashboard are way different from the Buttons we use in website currently. Your buttons don't show hover effect and slightly off from the mark of the theme as we use in terms of `primary` and `secondary` buttons.

### `Coming Soon` has to be improved in terms of colors

![image](https://user-images.githubusercontent.com/76521428/200590696-6685e9d9-5ae5-4b42-9894-b1f256cdf785.png)

The way `Coming Soon` has been indicated is not good from my perspective because it is not catching the reader's attention towards it. I would prefer to change it's background color or styling in such a way that you make it as eye-catching.

### Add some spacing between the components in mobile view

![image](https://user-images.githubusercontent.com/76521428/200591437-1fe9f4d7-7426-4ba4-9f97-a205bfa664f0.png)

![image](https://user-images.githubusercontent.com/76521428/200591512-2fe3ad06-054a-41ce-a312-9676aa2c3a03.png)

These are the 2 places I found to have some spacing missing between components in mobile view. Kindly add the respective spacing between them.

### Correct the z-index of Docs Nav

![image](https://user-images.githubusercontent.com/76521428/200592373-d5474a15-2fff-4634-ad78-75a127b0a12e.png)

In the mobile view, I found that the Docs Nav is getting overlapped by the Navbar of the mobile. Kindly look into the implementation of the `DocsNav` in the docs section and copy that here.

There are still many issues pending from my perspective inside the community section and will address them in upcoming reviews.

Barbanio commented:
@AceTheCreator How beautiful the ambassador page looks, I love the designs!

I have some suggestions:

- I think adding the link to the [FAQ](https://github.com/asyncapi/community/pull/361/files) document about the program is important. This document contains important things like the minimum number of contributions per year to be made by the ambassador. Perhaps we could add it in the description of the "AsyncAPI Ambassador Contributions" section.

- On the other hand, in the information about the ambassadors, we could add the [contributions](https://github.com/asyncapi/community/pull/475/files#diff-3972c4d1765c30a353c5f35562daa98dcd2bf7d3f2640d93f1ed03471777233e) (talks, articles...) they have made. And as you know, in the presentation cards of the ambassadors, we should put the information of Alejandra, Jesse Menning, and Hugo Guerrero, who are officially the ambassadors :) 

What do you think?

akshatnema commented:
![image](https://user-images.githubusercontent.com/76521428/200595607-f6477bb1-c084-4697-8a57-42b1b2c3e52c.png)

Also, I recently found that I'm getting certain console messages repeatedly while accessing certain pages. Kindly remove them.

quetzalliwrites commented:
> > 3. The `Getting Started` section looks like a mistaken duplicate of our current [AsyncAPI Docs Quickstart, "Getting Started."](https://www.asyncapi.com/docs/tutorials/getting-started) I do not think this title should be used for this section of the Community Docs. What is the purpose of this section?
> 
> The basis of this section is to contain docs like "How to become a TSC member", "How to create a new repository in the AsyncAPI organization" e.t.c

That is not the right kind of title for those docs. Please make updates based on the feedback items I made above, because I don't think I see you addressing my feedback changes :)

AceTheCreator committed:
enhanced community homepage

AceTheCreator committed:
adding contributors list

AceTheCreator committed:
added real ambassadors

AceTheCreator committed:
reused ambassadors data for contributors

AceTheCreator committed:
made required changes

AceTheCreator committed:
removed unneccessary console

AceTheCreator commented:
> Brilliant contributions @AceTheCreator 👏 , but also I would like to add few changes that I consider to be important. Right now, I have made review on the basis of UI preview, but will do the review in terms of code addition and implementation afterwards (as soon as I get free time). Here are some of the issues I'll like to entertain:
> 
> ### Not having proper color or background addition for cards
> ![image](https://user-images.githubusercontent.com/76521428/200588453-6fbc5a4b-033d-4f10-ac77-ba4a6b2da070.png)
> 
Removed this section. Seems unnecessary to me
> Tbh, if you are assuming them to be cards, you should have either a good background inside the each card or they should have a slight wide border so that user can identify that these cards are clickable and different from the background.
> 
> ### Buttons are not exactly in the way we use in website
> ![image](https://user-images.githubusercontent.com/76521428/200588993-229a242a-1462-45f5-a7c5-3e39b711bb04.png)
> 
> If we look into the Buttons you added in the Community Dashboard are way different from the Buttons we use in website currently. Your buttons don't show hover effect and slightly off from the mark of the theme as we use in terms of `primary` and `secondary` buttons.
> 
Fixed, but the few ones off in terms of color is simply cuz i want the bg to look off from what i'm trying to achieve
> ### `Coming Soon` has to be improved in terms of colors
> ![image](https://user-images.githubusercontent.com/76521428/200590696-6685e9d9-5ae5-4b42-9894-b1f256cdf785.png)
> 
> The way `Coming Soon` has been indicated is not good from my perspective because it is not catching the reader's attention towards it. I would prefer to change it's background color or styling in such a way that you make it as eye-catching.
> 
Removed this section
> ### Add some spacing between the components in mobile view
> ![image](https://user-images.githubusercontent.com/76521428/200591437-1fe9f4d7-7426-4ba4-9f97-a205bfa664f0.png)
> 
Fixed
> ![image](https://user-images.githubusercontent.com/76521428/200591512-2fe3ad06-054a-41ce-a312-9676aa2c3a03.png)
> 
> These are the 2 places I found to have some spacing missing between components in mobile view. Kindly add the respective spacing between them.
> 
Fixed
> ### Correct the z-index of Docs Nav
> ![image](https://user-images.githubusercontent.com/76521428/200592373-d5474a15-2fff-4634-ad78-75a127b0a12e.png)
> 
Had to hold on this... There's some discussion going on whether to add or remove the docs page under the community section
> In the mobile view, I found that the Docs Nav is getting overlapped by the Navbar of the mobile. Kindly look into the implementation of the `DocsNav` in the docs section and copy that here.
> 
Same as above
> There are still many issues pending from my perspective inside the community section and will address them in upcoming reviews.

Thanks for the feedback so far Askhat... Looking forward to more :)

AceTheCreator commented:
> @AceTheCreator How beautiful the ambassador page looks, I love the designs!
> 
> I have some suggestions:
> 
> * I think adding the link to the [FAQ](https://github.com/asyncapi/community/pull/361/files) document about the program is important. This document contains important things like the minimum number of contributions per year to be made by the ambassador. Perhaps we could add it in the description of the "AsyncAPI Ambassador Contributions" section.

I think this doc has to be published somewhere first, maybe in a blog. so that i can link to it from the ambassador page. wyt? 
> * On the other hand, in the information about the ambassadors, we could add the [contributions](https://github.com/asyncapi/community/pull/475/files#diff-3972c4d1765c30a353c5f35562daa98dcd2bf7d3f2640d93f1ed03471777233e) (talks, articles...) they have made. And as you know, in the presentation cards of the ambassadors, we should put the information of Alejandra, Jesse Menning, and Hugo Guerrero, who are officially the ambassadors :)
> 
Done!
> What do you think?

AceTheCreator commented:
@alequetzalli can you take a look at the updated PR? I made some improvement based on our last discussion.

AceTheCreator committed:
used server side rendering for cobntributors profile

AceTheCreator committed:
restructure contributor page

Barbanio commented:
> > * I think adding the link to the [FAQ](https://github.com/asyncapi/community/pull/361/files) document about the program is important. This document contains important things like the minimum number of contributions per year to be made by the ambassador. Perhaps we could add it in the description of the "AsyncAPI Ambassador Contributions" section.
> 
> I think this doc has to be published somewhere first, maybe in a blog. so that i can link to it from the ambassador page. wyt?
> 

I think it would be interesting to make a blog post about the program, but perhaps with more generic information. I think the most relevant data in this document, such as the obligations of the ambassadors or how to join the program, are merely informative for those who want to be ambassadors. I don't believe this is attractive to a more general reader. Still, I'm not sure. I would like to know what others think.

AceTheCreator commented:
> > > * I think adding the link to the [FAQ](https://github.com/asyncapi/community/pull/361/files) document about the program is important. This document contains important things like the minimum number of contributions per year to be made by the ambassador. Perhaps we could add it in the description of the "AsyncAPI Ambassador Contributions" section.
> > 
> > 
> > I think this doc has to be published somewhere first, maybe in a blog. so that i can link to it from the ambassador page. wyt?
> 
> I think it would be interesting to make a blog post about the program, but perhaps with more generic information. I think the most relevant data in this document, such as the obligations of the ambassadors or how to join the program, are merely informative for those who want to be ambassadors. I don't believe this is attractive to a more general reader. Still, I'm not sure. I would like to know what others think.
>

I agree with you on this

quetzalliwrites commented:
HUGE improvements @AceTheCreator, nice job!

I see 4 main bugs to address right now:
**(1)** **You've chosen some light grey gradient for some of the headers that are NOT visible.** Let's change these headers to have a readable hue, even if it's black and matches the other headers. (maybe go for a blue/purple gradient hue?)

<img width="400" alt="Screen Shot 2022-11-11 at 6 06 41 PM" src="https://user-images.githubusercontent.com/19964402/201451725-a7f695b7-3280-4a22-811a-864072f2bb8a.png">
<img width="400" alt="Screen Shot 2022-11-11 at 6 09 24 PM" src="https://user-images.githubusercontent.com/19964402/201451726-15422da0-7ff3-40e2-865e-640400ea0193.png">

(2) In **mobile**, the **Community goals** section has a bug; the events from the PREVIOUS `Meeting & Events` section are broken and cover all the text and pictures in **Community goals** area. 
<img width="300" alt="Screen Shot 2022-11-11 at 6 09 06 PM" src="https://user-images.githubusercontent.com/19964402/201451764-cd9e3fe2-f633-4fb9-9321-8632e2b340a7.png">

(3) In **mobile**, there is a HUGE blank space between the `Slack` and `newsletter` sections. Not sure what's causing that but it's def not a good thing. 
<img width="300" alt="Screen Shot 2022-11-11 at 6 10 08 PM" src="https://user-images.githubusercontent.com/19964402/201451911-c16a3818-1fed-4e48-8115-1b887730553d.png">

(4) In the **Community** menu item for the new `/Overview` page, we need to fix the punctuation. (I would have fixed it myself but I couldn't find the menu code in the PR. 😂   

**Let's change it to: `Connect, share, and learn.`**

<img width="300" alt="Screen Shot 2022-11-11 at 6 16 32 PM" src="https://user-images.githubusercontent.com/19964402/201451981-b6de93b2-3b53-4b09-895b-76e7826ac6b6.png">

quetzalliwrites commented:
Added a `dnm` label just in case 😄✌🏽

AceTheCreator committed:
enhancing community main page

AceTheCreator committed:
ambassador grid section fix

AceTheCreator committed:
fixed contributors responsiveness

AceTheCreator committed:
.

derberg commented:
I thought the idea of having a `community` branch is that as a result, we will get smaller focused-on-one-thing PRs 😅 

@AceTheCreator can you indicate what is the scope of this PR? @akshatnema already looks into code, so I'd like to see from UI perspective, but don't want to provide feedback on things that are planned for another PR.

You could update PR title and description 🙏🏼

AceTheCreator commented:
> I thought the idea of having a `community` branch is that as a result, we will get smaller focused-on-one-thing PRs 😅
> 
> @AceTheCreator can you indicate what is the scope of this PR? @akshatnema already looks into code, so I'd like to see from UI perspective, but don't want to provide feedback on things that are planned for another PR.
> 
> You could update PR title and description 🙏🏼

Yea, I think the reason I decided to go this route of adding other pages is cuz some reviewers think the missing pages weren't cool so decided to work on all the pages so that they can review the UI then I can focus on core functionality in the next PR

derberg commented:
Interesting. I'm not sure how others, but when I see 👇🏼 
![Screenshot 2022-11-15 at 17 23 18](https://user-images.githubusercontent.com/6995927/201972161-56243b54-1881-4c4f-9dc7-f4af333626af.png)

then PR goes to the bottom of the list of PRs for review. I saw your PR few days ago and that is what I did with my priority list 😆 

you need to find a good balance 😄

derberg commented:
> This PR shows all community sub-page's user interfaces which includes
/community/ambassador-programs
/community/contributors
/community/events

#### /community/ambassador-program

ok, so what is the scope of review here:
- text?
- these different colorful components? if they should be there or?
- how info about ambassadors is displayed?
- or the fact that you extended/modified original `AMBASSADORS_MEMBERS.json`

#### /community/events

it is a dead page

#### /community/contributors

here I do not even know where to start 😄 as it says contributors but on a list I see ambassadors. These are not the same, and the source of data is totally different.

---

I have no idea what is the scope of the review 😄

AceTheCreator commented:
> > This PR shows all community sub-page's user interfaces which includes
> > /community/ambassador-programs
> > /community/contributors
> > /community/events
> 
> #### /community/ambassador-program
> ok, so what is the scope of review here:
> 
> * text?
> * these different colorful components? if they should be there or?
> * how info about ambassadors is displayed?
> * or the fact that you extended/modified original `AMBASSADORS_MEMBERS.json`
> 
> #### /community/events
> it is a dead page
> 
> #### /community/contributors
> here I do not even know where to start 😄 as it says contributors but on a list I see ambassadors. These are not the same, and the source of data is totally different.
> 
> I have no idea what is the scope of the review 😄

The scope of review is design and text @derberg. The source of data is temporary for now... I had to use it just to have some output

AceTheCreator commented:
> > This PR shows all community sub-page's user interfaces which includes
> > /community/ambassador-programs
> > /community/contributors
> > /community/events
> 
> #### /community/ambassador-program
> ok, so what is the scope of review here:
> 
> * text?
> * these different colorful components? if they should be there or?
> * how info about ambassadors is displayed?
> * or the fact that you extended/modified original `AMBASSADORS_MEMBERS.json`
> 
> #### /community/events
> it is a dead page
> 
You sure? https://deploy-preview-1071--asyncapi-website.netlify.app/community/events
> #### /community/contributors
> here I do not even know where to start 😄 as it says contributors but on a list I see ambassadors. These are not the same, and the source of data is totally different.
> 
> I have no idea what is the scope of the review 😄

derberg commented:
I just noticed we did not have branch protection and code owners requirement for review for `community branch. Solved

quetzalliwrites commented:
> I just noticed we did not have branch protection and code owners requirement for review for `community branch. Solved

Yay!!!!!!!!!! 

This is what I was trying to solve when I added the DNM label. Of course adding branch protection is the appropriate way to fix this so now that Lucas made that improvement, I have removed the DNM label 😂

derberg commented:
> You sure? https://deploy-preview-1071--asyncapi-website.netlify.app/community/events

Interesting, 30min ago I was getting an internal server error. Thanks!

> The scope of review is design and text @derberg. The source of data is temporary for now... I had to use it just to have some output

The core of the issue is when you focus first on UI without real data you might design something that is not technically possible.

Examples:
- original https://github.com/asyncapi/community/blob/master/AMBASSADORS_MEMBERS.json file do not have info about the flag and bio. So do you propose extending it? Did you double-check with others if that makes sense, would ambassadors be willing to share nationality? does it bring value, or there idea is to share rather a language they can speak with - this could be useful I guess. But does it make sense with first iteration to already do such changes
-  in the events page I see different type of events, like webinar, conference. Where would you grab these from? There are different icons and stuff but the data that we have access to are not even used -> https://github.com/asyncapi/website/blob/master/config/meetings.json
- same with `contributors`, you operate on data that have no representation in reality, for example the flag that is there, these things are not available on profile level https://github.com/settings/profile. Also your design is not in sync with design from Maya -> https://github.com/asyncapi/website/issues/930
- also content on https://deploy-preview-1071--asyncapi-website.netlify.app/community/ambassador-program is not reflecting the reality described in https://github.com/asyncapi/community/blob/master/AMBASSADOR_ORGANIZATION.md. For example `Ambassadors are provided with a budget plan that they use for personal development tools, like books, courses, and other learning methods.` 👀 😆 

the point is that whatever we approve here, you will anyway rework in the next PR. You know what I mean?

> cuz some reviewers think the missing pages weren't cool

let's negotiate with these reviewers 😄 cause I bet that for the majority better is when PRs are super small 😄 (I bet $100 on it 😄 )

AceTheCreator commented:
@derberg @alequetzalli @akshatnema  I think the best way forward is to eventually merge this PR and start tackling the above request one after the other with a smaller PR :)

derberg commented:
hehe, nice try ☺️ 
Problem is that if you merge, then how can we provide feedback, add comments, etc. PR way is the best.

I know you will hate me for this comment but really the best way forward is to split it into 3 PRs 😅

quetzalliwrites commented:
Agreed... @AceTheCreator, you're so sneaky, trying to merge PRs before they're ready 😂 

Splitting this into smaller PRs (looks like Lukasz thinks 3 PRs are sufficient) is def the way to go 🙌🏽🙌🏽

I know it may seem "good enough" to use dummy data in the efforts toward getting this done faster, but like Lukasz notes above, you cannot know if your UI components will work with real-life data. And you shouldn't merge this to wait to find out how it works 😂

quetzalliwrites commented:
## **(1) Menu item bug**

Hey @AceTheCreator, there is still a punctuation issue in the menu item. You should copy/paste the text I gave you to help you avoid further punctuation mistakes 😄✌🏽  (you can also show me directly which file this change should be made to and I can make that commit.. just like I said above, I couldn't figure out which file to make that change to)

Let's fix the menu item to read: **Connect, share, and learn.**

 
<img width="400" alt="Screen Shot 2022-11-16 at 11 06 59 AM" src="https://user-images.githubusercontent.com/19964402/202270983-f36a862d-88ca-4b5e-a2d3-ce761a491eb2.png">

<img width="400" alt="Screen Shot 2022-11-16 at 11 07 16 AM" src="https://user-images.githubusercontent.com/19964402/202270986-e9daea11-621c-4e7c-be0c-0a997e51abd9.png">

##  **(2) odd spacing in mobile view**
There is still an odd spacing issue in mobile.

**Are you making sure to validate your changes in diff devices? Remember: you should always ensure your UI changes are responsive.** You need to use a _simulator_ or the _default web browser tools for device validation_.

<img width="600" alt="Screen Shot 2022-11-16 at 11 11 45 AM" src="https://user-images.githubusercontent.com/19964402/202272128-ec57fec3-dc47-4421-b40c-398fb7f5d311.png">

AceTheCreator commented:
I split this PR into smaller ones based on what @derberg and @alequetzalli suggested 

### Community landing page
https://github.com/asyncapi/website/pull/1119

### Ambassador page
https://github.com/asyncapi/website/pull/1123

### Events page
https://github.com/asyncapi/website/pull/1120

### Contributors page
https://github.com/asyncapi/website/pull/1122

I believe it would be easier now to make review much more efficient :) 

cc @derberg @alequetzalli @akshatnema @Barbanio

derberg commented:
Closing this PR as there were separate PRs processed
